README.ipynb

```diff
@@ -0,0 +1,45 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "67bb6fe5-38a1-472e-bf7c-9a300da42cb7",
+   "metadata": {},
+   "source": [
+    "# Before starting\n",
+    "\n",
+    "#### Directories and usage:\n",
```

Comment:
why did you decide on a notebook instead of a markdown?

Reply:
For two reasons:

1. A notebook containing markdown can directly display formatted markdown. With a markdown file you need to take actions to pull up the preview.
1. I think it It is possible to auto start a notebook when starting the service. Stoped half way through last time. I wanted to try something like this https://stackoverflow.com/a/71128495/2855718 . Will give it another try